# patch "libpurple/protocols/myspace/myspace.h"
#  from [3ff7642b4fa1325f420abb169ac45be9d51bdef5]
#    to [e2dde079de3df378e3bdb2aa3e91921788d7862f]
#
============================================================
--- libpurple/protocols/myspace/myspace.h	3ff7642b4fa1325f420abb169ac45be9d51bdef5
+++ libpurple/protocols/myspace/myspace.h	e2dde079de3df378e3bdb2aa3e91921788d7862f
@@ -41,7 +41,7 @@
 #include "notify.h"
 #include "plugin.h"
 #include "accountopt.h"
-#include "version.h"
+#include "libpurple/version.h"
 #include "cipher.h"     /* for SHA-1 */
 #include "util.h"       /* for base64 */
 #include "debug.h"      /* for purple_debug_info */
